### PR TITLE
[User Service & Question Service] Standardize endpoints

### LINKED ISSUE #32: [General] Standardise api endpoints

### PR DESCRIPTION
**Pull request by wweqg**
Close #32

### CODE REVIEW

File: user_service/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Replace <access_token> with a valid access token obtained by logging in using th
 
 ### Get User by ID 
 
-- **URL**: `/api/users/<str:username>`
+- **URL**: `/user-service/getUser/<str:username>`
```
Comment (izzahaj) [inline user_service/README.md]: (I'm assuming we should follow [REST resource-naming](https://restfulapi.net/resource-naming/)). Perhaps something like `/user-service/users/<str:username>` would adhere to the convention better. Same goes for the rest of the endpoints

**Review by izzahaj — COMMENTED** [review]
I think we agreed to stick to either the user or question service's naming conventions, (I'm assuming the changes are based off the names in the question service,) but I think we should follow RESTful naming conventions.

File: user_service/README.md (hunk 1 above)
Comment (wweqg) [reply user_service/README.md]: Updated

**Review by izzahaj — APPROVED** [review]
LGTM

**Comment by wweqg** [comment]
I will merge this after making the change on the question service endpoints

**Pull request merged**